buttons-examples/iOS/ios-button.md

```diff
@@ -24,444 +26,332 @@ There are four types of buttons:
-
-Before using the `MDCButtons` API to implement its types you must install `MDCButtons`. In your source files import the component, and then apply your theme:
-1. Install `MDCButtons`
-   * Use CocoaPods to install `MDCButtons`
-     1. Add the following line to your `Podfile`:
-       ```java
-      pod MaterialComponents/Buttons
-       ```
-     1. Run the installer:
-       ```bash
-       pod install
-       ```
-1. Import `MDCButtons` and MDC button theming and initialize `MDCButtons` using `alloc`/`init`. Initialize your theme  before applying it to your button.
-
-   _**Note** For more information about themes, go to the [Theming page](https://material.io/develop/ios/components/theming/) for iOS._
-    <!--<div class="material-code-render" markdown="1">-->
-   **Swift**
-   ```swift
-   import MaterialComponents.MaterialButtons
-   import MaterialComponents.MaterialButtons_Theming
-   /*...*/
-   let <local theme name> = <theme name>
-   let button = MDCButton()
-   ```
-   **Objective-C**
-   ```objc
-   #import "MaterialButtons.h"
-   #import <MaterialComponentsBeta/MaterialButtons+Theming.h>
-   /*...*/
-   <theme name> *<local theme name> = [[<theme name> alloc] init];
-   MDCButton *button = [[MDCButton alloc] init];
-   ```
-    <!--</div>-->
-
-    For our examples, we used the following theming values:
-
-    <!--<div class="material-code-render" markdown="1">-->
-   **Swift**
-   ```swift
-   let MyMaterialTheme = MDCContainerScheme()
-   ```
-   **Objective-C**
-   ```objc
-   MDCContainerScheme *MyMaterialTheme = []
-   ```
-    <!--</div>-->
-
-
-
-
-### Making buttons accessible
- 
-To help make your buttons usable to as many users as possible, apply the following:
-
-* Set an appropriate [`accessibilityLabel`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel) value if your button does not have a title or only has an icon:
-  <!--<div class="material-code-render" markdown="1">-->
-    **Objective-C**
-    ```objc
-    button.accessibilityLabel = @"Create";
-    ```
-    **Swift**
-    ```swift
-    button.accessibilityLabel = "Create"
-    ```
-  <!--</div>-->
+### Install `MDCButton`
 
-* Set the minimum [visual height to
-36 and miniumum visual width to 64](https://material.io/design/components/buttons.html#specs)
-  <!--<div class="material-code-render" markdown="1">-->
-    **Objective-C**
+<details><summary><b>Expand for installation instructions for <code>MDCButton</code></b></summary>
 
-    ```objc
-    button.minimumSize = CGSizeMake(64, 36);
-    ```
+<br>
 
-    **Swift**
+`MDCButton` is used to implement all four Material Buttons. In order to use `MCDButton`, do the following:
 
-    ```swift
-    button.minimumSize = CGSize(width: 64, height: 48)
+1. Install with Cocoapods
+    Add the following line to your `Podfile`:
+
+    ```
+    pod MaterialComponents/Buttons
+    ```
+    
+    Run the installer:
+    
+    ```
+    pod install
     ```
-  <!--</div>-->
```

Comment:
We need to keep the <!--<div>--> comments because they render the radio selectors currently used to switch between swift and obj-c code:

https://material.io/develop/ios/components/buttons/

Reply:
Done.